python/llm/example/CPU/HF-Transformers-AutoModels/Model/mistral/README.md

```diff
@@ -19,6 +19,7 @@ pip install --pre --upgrade bigdl-llm[all] # install the latest bigdl-llm nightl
 # Refer to https://huggingface.co/mistralai/Mistral-7B-v0.1#troubleshooting, please make sure you are using a stable version of Transformers, 4.34.0 or newer.
 pip install transformers==4.34.0
 ```
+**Important: According to [Mistral Troubleshooting](https://huggingface.co/mistralai/Mistral-7B-v0.1#troubleshooting), please make sure you have installed `transformers==4.34.0` following above commands.**
```

Comment:
move the notice to the requirements section

Reply:
> move the notice to the requirements section

Have moved.